tests/api/Jalasoft.TeamUp.Projects.API.Tests/PatchInvitationByIdTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿namespace Jalasoft.TeamUp.Projects.API.Tests
+{
+    using System;
+    using System.Collections.Generic;
+    using Jalasoft.TeamUp.Projects.API.Controllers;
+    using Jalasoft.TeamUp.Projects.API.Tests.Utils;
+    using Jalasoft.TeamUp.Projects.Core.Interfaces;
+    using Jalasoft.TeamUp.Projects.Models;
+    using Jalasoft.TeamUp.Projects.ProjectsException;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Moq;
+    using Xunit;
+
+    public class PatchInvitationByIdTests
+    {
+        private readonly Mock<IInvitationsService> mockService;
+        private readonly DefaultHttpContext mockHttpContext;
+        private readonly PatchInvitationById updateInvitation;
+
+        public PatchInvitationByIdTests()
+        {
+            this.mockService = new Mock<IInvitationsService>();
+            this.mockHttpContext = new DefaultHttpContext();
+            this.updateInvitation = new PatchInvitationById(this.mockService.Object);
+        }
+
+        [Theory]
+        [InlineData(@"[
+        {""op"" : ""replace"", ""path"" : ""/status"", ""value"" : ""Accepted""}
+        ]")]
```

Comment:
if you have only one use case (inline data) this could be a Fact instead of a Theory
add more test cases (edge scenarios) which I'd do, or make this as Fact (which I'd avoid)

Reply:
done